### PR TITLE
chore: Sync conflicting translations

### PR DESCRIPTION
**Pull request by joeyparrish**
This imports newly-conflicting translations for review.  We have conflicts in:

 - Catalan (ca)
 - German (de)
 - Spanish (Spain, es)
 - Spanish (Latin American, es-419)
 - French (fr)
 - Polish (pl)
 - Portuguese (Portugal, pt-PT)
 - Portuguese (Brazil, pt-BR)

### CODE REVIEW

**Comment by joeyparrish** [comment]
@avelad for Spanish, @tykus160 for Polish

Still looking for native speakers to review other languages: Catalan, German, French, Portuguese.

**Comment by joeyparrish** [comment]
@sanao, you contributed our last updates in French.  Can you review these changes please?

File: ui/locales/de.json
Comment (mthie) [inline ui/locales/de.json:22]: The previous translation with `Schleife` or `Dauerschleife` (aka endless loop) is better. `Wiederholen` is a word for `repeat`.

Comment (joeyparrish) [reply ui/locales/de.json:22]: Vielen Dank!

File: ui/locales/pl.json
Comment (tykus160) [inline ui/locales/pl.json:42]: I think it will be better to stick with "Wymuszone" form, as it's plural (the same as Polish word for subtitles, "napisy").
```suggestion
  "SUBTITLE_FORCED": "Wymuszone",
```

File: ui/locales/pl.json
Comment (tykus160) [inline ui/locales/pl.json:43]: As this label is stitched to the language (and optionally codec) I'd suggest to keep just "przestrzenny" adjective.
```suggestion
  "SURROUND": "Przestrzenny",
```

**Review by tykus160 — COMMENTED** [review]
2 minor comments, otherwise it's good.

**Review by Boubalou — APPROVED** [review]
French is looking good for me!

**Comment by joeyparrish** [comment]
Still looking for native speakers to review: Catalan, Portuguese.

File: ui/locales/fr.json
Comment (Iragne) [inline ui/locales/fr.json:1]: My view on this translation 
"DOWNLOAD_VIDEO_FRAME": "Enregistrer l'image vidéo",
I'm seeing it more like that "Télécharger l'image de la vidéo"

**Comment by Iragne** [comment]
> French is looking good for me!

I would like to propose that https://github.com/shaka-project/shaka-player/pull/7262/files#r1756426907

File: ui/locales/fr.json
Comment (Boubalou) [reply ui/locales/fr.json:1]: Good point, I only looked into the proposed changes, not the whole file. 

For this one, if the context of the label is to download the whole video, it could be more like: `Télécharger la vidéo`.

If its to download the frames, that would be `Télécharger les images de la vidéo`.

File: ui/locales/de.json
Comment (joeyparrish) [reply ui/locales/de.json]: Reverted.

File: ui/locales/es.json
Comment (joeyparrish) [reply ui/locales/es.json]: Reverted.

File: ui/locales/fr.json
Comment (joeyparrish) [reply ui/locales/fr.json:1]: Since it's a single frame, going with "Télécharger l'image de la vidéo"

File: ui/locales/pl.json
Comment (joeyparrish) [reply ui/locales/pl.json]: Reverted.

File: ui/locales/pl.json
Comment (joeyparrish) [reply ui/locales/pl.json]: Reverted.

**Comment by joeyparrish** [comment]
Now I need one final approving review from a maintainer to merge this.

**Review by avelad — APPROVED** [review]

**Comment by joeyparrish** [comment]
I will push the community edits back upstream to Google's Translation Console.

**Pull request merged**
